cmake/onnxruntime_webassembly.cmake

```diff
@@ -61,6 +61,12 @@ else()
   set_property(TARGET onnxruntime_webassembly APPEND_STRING PROPERTY LINK_FLAGS " -s ASSERTIONS=0 -s SAFE_HEAP=0 -s STACK_OVERFLOW_CHECK=0 -s DEMANGLE_SUPPORT=0")
 endif()
 
+# Set link flag for onnxruntime-extensions because now it contains submodules using exceptions
+# Would remove this link flag when all submodules have supported disabling c++ exceptions
+if (onnxruntime_USE_EXTENSIONS)
```

Comment:
And disable-exception flag?

Reply:
Hi @wenbingl and @fs-eire , according to your advice, I have added a flag to build and enable linking exceptions library when build with `--disable_exceptions`. Please help review, thanks.